apps/base/models.py

```diff
@@ -14,3 +19,18 @@ class TimeStampedModel(models.Model):
     class Meta:
         abstract = True
         app_label = 'base'
+
+
+def extra_args(fragment_name, *args, **kwargs):
+    def extra_args_inner1(f, *args, **kwargs):
+        def extra_args_inner2(sender, instance, **kwargs):
+            f(sender, instance, fragment_name=fragment_name, **kwargs)
+        return extra_args_inner2
+    return extra_args_inner1
+
+
+def create_post_model_field(sender, instance, fragment_name, **kwargs):
+    if getattr(instance, "_original_" + str(fragment_name)) is False:
```

Comment:
Convert `str(fragment_name)` -> `fragment_name`. There is no need to use `str()`